assets/models/space/PlaceholderAdvancedSpaceship.tscn

```diff
@@ -0,0 +1,87 @@
+[gd_scene load_steps=12 format=3 uid="uid://bdvhu67uu45ej"]
+
+[ext_resource type="Texture2D" uid="uid://brqk1odkw0fy3" path="res://assets/textures/space/Spaceship_albedo_roughness.png" id="1_7u0ip"]
+[ext_resource type="Texture2D" uid="uid://cda1j1liv0bks" path="res://assets/textures/space/Spaceship_emit.png" id="2_sojrh"]
+[ext_resource type="ArrayMesh" uid="uid://ck75ncsm5nox5" path="res://assets/models/space/PlaceholderAdvancedSpaceship.mesh" id="3_g47x4"]
+[ext_resource type="Texture2D" uid="uid://kloyvken30uu" path="res://assets/textures/menu_backgrounds/SingleClouds.png" id="6_dxu3k"]
+[ext_resource type="Texture2D" uid="uid://dhad2rl5f4478" path="res://assets/textures/menu_backgrounds/SingleClouds_Normal.png" id="7_sct06"]
+
+[sub_resource type="StandardMaterial3D" id="StandardMaterial3D_aurqk"]
+albedo_texture = ExtResource("1_7u0ip")
+metallic = 0.8
+metallic_texture = ExtResource("1_7u0ip")
+metallic_texture_channel = 3
+roughness = 0.7
+roughness_texture = ExtResource("1_7u0ip")
+roughness_texture_channel = 3
+emission_enabled = true
+emission = Color(0.0588235, 0.0588235, 0.0588235, 1)
+emission_energy_multiplier = 2.0
+emission_texture = ExtResource("2_sojrh")
+
+[sub_resource type="Curve" id="Curve_tq58m"]
+resource_local_to_scene = true
```

Comment:
This file also has local to scene stuff, I'll stop commenting on this now as it seems a ton of files have these. So could you please go through your own changes and look for the rest?

Reply:
I think I started making these changes before I was clear about when to use that. I've realized that it's much more efficient for removing them to just search for them all, too.